loopchain/peer/consensus_siever.py

```diff
@@ -23,144 +26,97 @@ class ConsensusSiever(ConsensusBase):
-                    self._blockmanager.get_blockchain().set_last_commit_state(
-                        self._block.height, self._block.commit_state)
-                else:
-                    # 검증 받을 블록의 hash 를 생성하고 후보로 등록한다.
-                    # logging.warning("add unconfirmed block to candidate blocks")
-                    self._block.generate_block(self._candidate_blocks.get_last_block(self._blockchain))
-
-                if block_is_verified:
-                    self._block.sign(ObjectManager().channel_service.peer_auth)
-                    self._candidate_blocks.add_unconfirmed_block(self._block)
-
-                    # 새로운 Block 을 생성하여 다음 tx 을 수집한다.
-                    self._gen_block()
-                else:
-                    failed_block = self._block
-                    self._gen_block()
-                    self.__throw_out_block(failed_block)
-
-            self._makeup_block()
-
-            # 다음 검증 후보 블럭이 있는지 확인한다.
-            candidate_block = self._candidate_blocks.get_candidate_block()
-            peer_manager = ObjectManager().channel_service.peer_manager
-
-            if candidate_block is not None:
-                # 있으면 해당 블럭을 broadcast 하여 Peer 에게 검증을 요청한다.
-                self._current_vote_block_hash = candidate_block.block_hash
-                logging.info(f"candidate block height: {candidate_block.height}")
-                logging.info("candidate block hash: " + self._current_vote_block_hash)
-
-                util.logger.spam(f"consensus_siever:consensus try peer_manager.get_next_leader_peer().peer_id")
-                candidate_block.next_leader_peer = peer_manager.get_next_leader_peer().peer_id
-
-                # 생성된 블럭을 투표 요청하기 위해서 broadcast 한다.
-                self._blockmanager.broadcast_send_unconfirmed_block(candidate_block)
-
-                # broadcast 를 요청했으면 다음 투표 block 이 있는지 계속 검사하기 위해 return 한다.
-                return result
-            elif self._block is not None and \
-                    not conf.ALLOW_MAKE_EMPTY_BLOCK and \
-                    (self._block.prev_block_confirm is True) and \
-                    (self._block.confirmed_tx_len == 0):
-                # 검증할 후보 블럭이 없으면서 이전 블럭이 unconfirmed block 이면 투표가 담긴 빈 블럭을 전송한다.
-                self._block.prev_block_hash = confirmed_block.block_hash
-                self._block.block_type = BlockType.vote
-
-                self._block.next_leader_peer = peer_manager.get_next_leader_peer().peer_id
-
-                self._blockmanager.broadcast_send_unconfirmed_block(self._block)
-
-                self._stop_gen_block()
-                util.logger.spam(f"consensus_siever:consensus channel({self._channel_name}) "
-                                 f"\ntry ObjectManager().peer_service.rotate_next_leader({self._channel_name})")
-
-                ObjectManager().channel_service.state_machine.turn_to_peer()
-
-        return result
+        block_builder = self._makeup_block()
```

Comment:
block_builder is shadow name from `loopchain/blockchain/blocks/block_builder`.
ref : https://stackoverflow.com/q/20125172

Reply:
Fixed it by modifying import packages.